wiki/Tournaments/GO/3/en.md

```diff
@@ -67,8 +65,9 @@ The Game\_Osu Tourney 3 is run by various community members.
   - [Game\_Osu (EN)](https://www.twitch.tv/gameosu)
   - [Sardoche's Twitch channel](https://www.twitch.tv/sardoche) (Finals and Grand Finals Livestream)
   - [Game\_Osu (FR)](https://www.twitch.tv/gameosu_fr)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vQkxCyBeU6EG2qiHd8ZMrrHYiEm7wxjXCCl73eH8Ttoh7_n9Gz6XfDsm8_tg2Proo9hQR11PATPBGGn/pubhtml)**
 
------------------------
+---
 
 ## Participants
```

Comment:
not sure why flag is its own column in this table, you could do it like everywhere else (Flag + space + Player)